psqlgraph/query.py

```diff
@@ -155,10 +159,9 @@ def not_ids(self, ids):
         """
 
         _id = self.entity().node_id
-        if hasattr(ids, '__iter__'):
-            return self.filter(not_(_id.in_(ids)))
-        else:
-            return self.filter(not_(_id == str(ids)))
```

Comment:
feel like this is covering a case when your `ids == 1` for example

Reply:
The `in_` should now cover this scenario also

Reply:
`g.nodes().not_ids(1)`? or `g.nodes().not_ids(None)`?